apps/guardian-ui/src/utils/env.ts

```diff
@@ -1,15 +1,16 @@
-export function getEnv() {
-  // This is a hack to get around react-scripts inlining environment variable
-  // checks. Boolean conditionals get reduced to `!0` and `!1` which breaks
-  // environment variable replacements from `replace-react-env.js`.
-  // Note that if you actually WANT that inlining, you should use `process.env`
-  // directly in your code, NOT `getEnv()`
-  // See https://create-react-app.dev/docs/adding-custom-environment-variables/ for more info
-  // See https://github.com/fedimint/ui/issues/224 for the long-term fix for configuration
-  return JSON.parse(
-    JSON.stringify({
-      FM_CONFIG_API: process.env.REACT_APP_FM_CONFIG_API,
-      TOS: process.env.REACT_APP_TOS,
-    })
-  );
+type EnvVars = {
+  fm_config_api?: string;
+  tos?: string;
+};
+
+export async function getEnv() {
+  const response = await fetch('config.json');
+  if (!response.ok) {
+    throw new Error('Could not find config.json');
+  }
+  const config: EnvVars = await response.json();
+  if (config.fm_config_api === 'config api not set' || !config.fm_config_api) {
+    throw new Error('Config API not set in config.json');
+  }
+  return config;
```

Comment:
This type def and function are now in two places, here and `GuardianApi`

Reply:
Good catch. Will refactor.